dist-persist/wbstack/src/Logging/CustomLogger.php

```diff
@@ -51,11 +51,33 @@ private function doLog( $level, $message, $context ) {
             ];
         }
 
-        $payload[ '@type' ] = 'type.googleapis.com/google.devtools.clouderrorreporting.v1beta1.ReportedErrorEvent';
+        /**
+         * log levels according to PSR-3: https://www.php-fig.org/psr/psr-3/#11-basics
+         * debug, info, notice, warning, error, critical, alert, emergency
+         */
+        $logLevelsCloudErrorReporting = ['error', 'critical', 'alert', 'emergency'];
+
+        if (in_array($level, $logLevelsCloudErrorReporting)) {
+            $payload[ '@type' ] = 'type.googleapis.com/google.devtools.clouderrorreporting.v1beta1.ReportedErrorEvent';
+        }
+
         $payload[ 'severity' ] = $level;
         $payload[ 'serviceContext' ] = [
             'service' => 'WBaaS MediaWiki',
-            'version' => '1.0.0'
+            'version' => '2.0.0'
```

Comment:
What would we think about using the `MW_VERSION` constant here as well? 🤔 it makes it show up separately in GCP:

<img width="1346" height="478" alt="Image" src="https://github.com/user-attachments/assets/f702ae61-19a8-4c2e-9657-eabc89f3540a" />

Reply:
I think using `MW_VERSION` makes sense, a lot more than `1.0.0` and `2.0.0` does anyway :sweat_smile: